webapp/channels/src/components/user_settings/notifications/user_settings_notifications.tsx

```diff
@@ -228,7 +228,7 @@ function getDefaultStateFromProps(props: Props): State {
     };
 }
 
-const Input = (props: InputProps<MultiInputValue, true>) => {
+export const CreatableReactSelectInput = (props: InputProps<MultiInputValue, true>) => {
```

Comment:
what is the logic behind this name change?

Reply:
This is a new exported item and there are other existing Input exports, so exporting this was adding to the confusion, so I renamed it to something more specific.